STARTERKIT/components/base/root/base-root.twig

```diff
@@ -14,7 +14,7 @@
 <html {{ html_attributes }}>
   <head>
     {% block head %}
-      <title>{{ title }}</title>
+      <title>{{ title|raw }}</title>
```

Comment:
Can't figure out how this change is related to tabs... please explain
And adding raw filter is really a bad thing to do - it should come with a comment explaining why the value is already escaped.